_includes/selector.html

```diff
@@ -379,30 +362,26 @@
             active_python_ver: "3.10",
             active_cuda_ver: "11.8",
             active_pip_cuda_ver: "12.0",
-            active_os_ver: "Ubuntu 22.04",
             active_method: "Conda",
             active_release: "Stable",
             active_arch: "amd",
-            active_img_type: "core",
+            active_img_type: "Base",
             active_img_loc: "NGC",
             active_packages: ["Standard"],
-            active_img_options: ["notebooks"],
             active_additional_packages: [],
 
             // all possible values
             python_vers: ["3.9", "3.10"],
             cuda_vers: ["11.2", "11.4", "11.5", "11.8", "12.0"],
             pip_cuda_vers: ["11.2 - 11.8", "12.0"],
-            os_vers: ["Ubuntu 20.04", "Ubuntu 22.04", "CentOS 7", "Rocky Linux 8"],
             methods: ["Conda", "pip", "Docker", "Source"],
             releases: ["Stable", "Nightly"],
             arches: ["amd", "arm"],
             img_loc: ["NGC", "Dockerhub"],
-            img_types: ["core", "standard"],
+            img_types: ["Base", "Notebooks"],
             packages: ["Standard", "cuDF", "cuML", "cuGraph", "cuSpatial", "cuXFilter", "cuSignal", "cuCIM"],
             pip_packages: ["cuDF", "dask-cuDF", "cuML", "cuGraph", "cuSpatial", "cuCIM"],
-            additional_packages: ["Dask SQL", "JupyterLab", "Plotly Dash", "Graphistry", "PyCaret", "TensorFlow", "Xarray-Spatial"], // , "RTXpy" will return later
-            img_options: ["notebooks", "development"],
+            additional_packages: ["Dask SQL", "JupyterLab", "Plotly Dash", "Graphistry", "TensorFlow", "Xarray-Spatial"], // , "RTXpy" will return later
```

Comment:
Rename `Dask SQL` to `Dask-SQL` (or `dask-sql`) anywhere that occurs.
```suggestion
            additional_packages: ["Dask-SQL", "JupyterLab", "Plotly Dash", "Graphistry", "TensorFlow", "Xarray-Spatial"], // , "RTXpy" will return later
```

Reply:
Resolved in https://github.com/rapidsai/docs/pull/414/commits/cb8e582dd6c332903e2cff27ed440967d7d0f222